runtime/src/bank.rs

```diff
@@ -1370,7 +1371,7 @@ impl Bank {
                 &signature_count_buf,
             ])
         } else {
-            self.parent_hash
+            hash(self.parent_hash.as_ref())
```

Comment:
@aeyakovenko I'm adding blockhash into the mix for bank hashes. 
For the else condition here should I still hash the parent's delta and parent's signature_counts?